installer/dbscript/dbscript-2.sql

```diff
@@ -4258,7 +4258,10 @@ VALUES ('apiv2_time_time_sheet_config', 'API-v2 Time - Time Sheet Start Day Conf
        ('apiv2_time_timesheet_comment', 'API-v2 Time - Timesheet comment', 1, 1, 1, 0),
        ('apiv2_time_config_time_format', 'API-v2 Time - Time Config Format', 1, 0, 0, 0),
        ('apiv2_time_validation_project_activity_name', 'API-v2 Time - Project Activity Name Validation', 1, 0, 0, 0),
-       ('apiv2_time_project_activity_unique_validation', 'API-v2 Time - Project Activity Unique Validation', 1, 0, 0, 0);
+       ('apiv2_time_project_activity_unique_validation', 'API-v2 Time - Project Activity Unique Validation', 1, 0, 0, 0),
+       ('apiv2_time_employee_timesheets', 'API-v2 Time - Employee Timesheets', 1, 0, 0, 0);
```

Comment:
```suggestion
       ('apiv2_time_employee_timesheets', 'API-v2 Time - Employee Timesheets', 1, 1, 1, 0);
```
check `apiv2_time_my_timesheet`

Reply:
done